app/views/audit_mailer/summary.text.erb

```diff
@@ -0,0 +1,13 @@
+<%= _("Foreman Audit summary") %>
+=================================
+<% if @audits.size > 0 %>
+  <%= _('Displaying %{num_audits} audits of %{total_audits} audits') % {:num_audits => @limit, :total_audits => @count} %>
```

Comment:
Use n_() and remove the first "audits" word so there aren't two plural/singular variables.